charms/openfga-k8s/src/charm.py

```diff
@@ -200,25 +194,33 @@ def _on_update_status(self, _):
         """Update the status of the charm."""
         self._ready()
 
+    def _create_token_secret(self, event):
+        if not self._state.is_ready():
+            event.defer()
+            logger.warn("state not ready")
+            return
+
+        if not self._state.token_secret_id:
+            token = secrets.token_urlsafe(32)
```

Comment:
What is this 32 bit? Should we make it a constant so it has a name?

Reply:
i don't know.. it think this is ok